python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -784,6 +803,10 @@ def load_context(self, params: PipesParams) -> Iterator[PipesContextData]:
 class PipesDbfsMessageWriter(PipesBlobStoreMessageWriter):
     """Message writer that writes messages by periodically writing message chunks to a directory on DBFS."""
 
+    def __init__(self, spark: Any, *, interval: float = 10):
+        super().__init__(interval=interval)
+        self.spark = spark
```

Comment:
We need to pass this into the message writer because we can't import it. `spark` is injected into the namespace of the entry point for a databricks job, but it is not globally injected, and `spark` is not available on `sys.path`.

Reply:
Is `spark` the spark session or something else?

Reply:
It is the spark session and I realized that it can be obtained through importing `SparkSession` from `pyspark.sql` and calling `getActiveSession`, so I've removed the arg where it gets passed in.